File: Python/beginner/1151.py
# https://judge.beecrowd.com/pt/problems/view/1151
# Fibonacci Fácil

# Requesting
usr_input = int(input())

# Starting another variables
control1 = 0
control2 = 1
aux = 0

# Testing input
if usr_input == 0:
    exit()

# First Print
print(f'{aux} ', end='')
aux = control1 + control2

# Iterating for the next values
for i in range(1, usr_input, 1):
    # Testing if it's the last
    if i == usr_input - 1:
        print(f'{aux}')
    else:
        print(f'{aux} ', end='')
        aux = control1 + control2
        control1 = control2
        control2 = aux

# By Fernando Serra
# https://github.com/fernandosserra

File: C/beginner/1151.c
    // https://judge.beecrowd.com/pt/problems/view/1151
    // Fibonacci Fácil

    // Libraries
    #include <stdio.h>

    // Main Function
    int main(){
        
        // Variables
        int usr_input, control_1, control_2, aux_value;

        control_1 = 0;
        control_2 = 1;
        aux_value = 0;

        // Getting values
        scanf("%d", &usr_input);

        // Testing input
        if (usr_input == 0){
            return 0;
        }    
        // First Print
        printf("%d ", aux_value);
        aux_value = control_1 + control_2;

        // Iterating
        for (int i = 1; i < usr_input; i++){
                        
            if (i == usr_input - 1){
                printf("%d\n", aux_value);
            } else {
                printf("%d ", aux_value);
                aux_value = control_1 + control_2;        
                control_1 = control_2;
                control_2 = aux_value;
            }                
        }
        
        return 0;

    }

    // By Fernando Serra & Franklin Pereira
    // https://github.com/fernandosserra
    // https://github.com/Hyuse98